src/background/hypothesis-chrome-extension.js

```diff
@@ -161,6 +161,11 @@ export default function HypothesisChromeExtension({
     return activeState;
   }
 
+  /**
+   *
+   * @param {number} tabId
+   * @param {string} url
+   */
```

Comment:
```suggestion
  /**
   * @param {number} tabId
   * @param {string} url
   */
```

Stylistic nitpick - for JSDoc that has no description but only param types, avoid extra blank lines.

Reply:
I think I fixed this in another PR...